### DISCUSSION TITLE
Why does conflict arise before merging branches?

### DISCUSSION THREAD

**Discussion by shettyvarshaa** [community/community#63176] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I have seen conflict arise while I try to merge branches and I do not know how to resolve that

**Comment by Ananyapk25** [comment]
conflict arises when you try two branches have changes in the same line of code and you either have to choose both these changes or any one of them

**Comment by queenofcorgis** [comment]
GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
